src/main/java/com/campool/service/BookingService.java

```diff
@@ -86,4 +87,13 @@ public int getCountByDate(LocalDate startDate, LocalDate endDate) {
         return bookingMapper.findStatesByIdAndStatus(id, BookingStatus.PAYMENT_COMPLETED);
     }
 
+    public void validateCancelRequest(CancelPaymentRequest request, String id) {
+        BookingInfo bookingInfo = getBookingInfoById(request.getMerchantUid());
+        if (bookingInfo.getAmount() != request.getPaidAmount()
+                || bookingInfo.getRentalStatus() != RentalStatus.TRADEABLE
+                || !bookingInfo.getUserId().equals(id)) {
```

Comment:
이런 식으로 하면 좀 더 가독성이 좋아질 것 같긴 하네요

```suggestion
        if (isNotValidAmount(...) || isNotTradeable(...) || isNotRentalOwner(...)) {
```

Reply:
리뷰반영 했습니다!